### Geofrenzy GfDwellProfile message
#
# A dwell message specifies a current entitlement in relation to the tracked
# location.
#
# The profile dwell ... 
#
Header        header      # Standard ROS header
GfEntProfile  entitlement # Dwell entitlement
